docs/tutorials/vertical-fl.md

```diff
@@ -0,0 +1,113 @@
+
+## Infrastructure
+First step towards successfully running VFL example is to provision an infrastructure. In order to do so, please navigate to [quickstart](../quickstart.md) and use single-button deployment for vnet infrastructure deployment. This is necessary in order for nodes to be able to communicate.
+
+## Install the required dependencies
+
+You'll need python to submit experiments to AzureML. You can install the required dependencies by running:
+
+```bash
+conda env create --file ./examples/pipelines/environment.yml
+conda activate fl_experiment_conda_env
+```
+
+Alternatively, you can just install the required dependencies:
+
+```bash
+python -m pip install -r ./examples/pipelines/requirements.txt
+```
+
+## Data provisioning
+The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading data that are needed for a given example.
+
+> Note: This will split the data such that each contributor owns its portion of the features and host own only the labels
+
+### CCFRAUD
+
+Please follow steps in [CCFRAUD - Add your Kaggle credentials to the workspace key vault](../real-world-examples/ccfraud.md#Add-your-Kaggle-credentials-to-the-workspace-key-vault). Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example CCFRAUD_VERTICAL`**).
+
+### MNIST
+
+This can all be performed with ease using a data provisioning pipeline. To run it follow these steps:
+
+1. If you are not using the quickstart setup, adjust the config file  `config.yaml` in `examples/pipelines/utils/upload_data/` to match your setup.
+
+2. Submit the experiment by running:
+
+   ```bash
+   python ./examples/pipelines/utils/upload_data/submit.py --example MNIST_VERTICAL --workspace_name "<workspace-name>" --resource_group "<resource-group-name>" --subscription_id "<subscription-id>"
+   ```
+
+   > Note: You can use --offline flag when running the job to just build and validate pipeline without submitting it.
+
+    :star: you can simplify this command by entering your workspace details in the file `config.yaml` in this same directory.
+
+:warning: Proceed to the next step only once the pipeline completes. This pipeline will create data in 3 distinct locations.
+
+## Model preparation for VFL
+The model in VFL can be either deployed by splitting it between the host and contributors, also referred to as **split learning**, or hosting whole model on contributor while host provides only aggregation function. Our examples relies on former as we believe this can better demonstrate capabilities of VFL on AzureML the proposed solution can be easily altered to the latter case.
```

Comment:
Why can split learning better demonstrate the capabilities of VFL on Azure ML better than the other approach?

"can be _easily_ altered": _easily_ can be quite subjective. If it's objectively easy maybe we don't need to show an example, but shouldn't we at least write down what steps need to be taken?